service/api_logic/templates/reset_password_email.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+    <body style="font-family: Arial, sans-serif; line-height: 1.6; color: #333; background-color: #f9f9f9; padding: 20px;">
+        <div style="max-width: 600px; margin: 0 auto; background: white; border: 1px solid #ddd; border-radius: 10px; padding: 20px;">
+            <h2 style="color: #4CAF50; text-align: center;">Password Reset Request</h2>
+            <p>Dear <strong>{{ username }}</strong>,</p>
+            <p>You have requested to reset your password. To do so, click the button below:</p>
+            <div style="text-align: center; margin: 20px 0;">
+                <a href="{{ reset_url }}" style="display: inline-block; background-color: #4CAF50; color: white; text-decoration: none; padding: 10px 20px; border-radius: 5px; font-size: 16px;">Reset Your Password</a>
+            </div>
+            <p>If the button above doesn't work, copy and paste the following link into your browser:</p>
```

Comment:
```suggestion
            <p>If the button above does not work, follow the link below:</p>
```

"doesn't" is informal. we should be using formal English when contacting our users